remediation/self-node-remediation/how-it-works.md

```diff
@@ -36,3 +36,27 @@ If there's an api-server failure, which means that all nodes can't access the ap
 It would create an unnecessary reboot storm in the cluster.
 
 ![self-node-remediation-api-server-failure.png](../../images/self-node-remediation-api-server-failure.png)
+
+## Control plane fencing
+Control plane fencing can be divided into two primary scenarios.
+
+### API Server Connectivity
+In this scenario, the control plane node to be remediated is not isolated. It can be directly connected to the API Server, or it can be indirectly connected to the API Server through worker nodes or control-plane nodes, that are directly connected to the API Server.
```

Comment:
You might consider to reduce some repetitions with a formula like:
"It can be connected to the API Server directly or indirectly through workers or control-plane nodes that are connected to the API Server"